fixed issues to make it work

rickjohnston opened the pull request.

rickjohnston committed:
fixed issues to make it work

lakshmi-kannan commented:
@rickjohnston My apologies. I didn't look at this PR until now. I'll try to run it and see if works.

mattmaltieri committed:
Updated generate_access_token.py

Made some changes to accomodate what appears to be a new format for the oauth URL, removed some needless comments and debug-type lines that were print() -ing data that was simply cluttering the script and output ... and ensured it works again!
